WTWSMS/localisation/VIET_modifiers.csv

```diff
@@ -295,4 +295,11 @@ VIET_province_strong_lads;Strong Lads;;;;;;;;;;;;;x
 VIET_province_strong_lads_desc;The young men of this province have achieved some level of renown for their bravery and skill in war.;;;;;;;;;;;;;x
 VIET_going_to_Scarborough_Fair;Scarborough Fair;;;;;;;;;;;;;x
 VIET_going_to_Scarborough_Fair_desc;This character has heard that great song, Scarborough Fair, and he or she has been enlightened by it. Onwards, as they say, to Scarborough Fair!;;;;;;;;;;;;;x
-VIET_erected_high_cross;Erected High Cross;;;;;;;;;;;x
\ No newline at end of file
+VIET_erected_high_cross;Erected High Cross;;;;;;;;;;;x
+VIET_opinion_refused_muafname;Refused muafname;;;;;;;;;;;x
+VIET_opinion_granted_small_muafname;Granted small muafname;;;;;;;;;;;x
+VIET_opinion_granted_big_muafname;Granted big muafname;;;;;;;;;;;x
+opinion_ugly_omelette_cook;Ugly omelette cook;;;;;;;;;;;x
+VIET_remove_kebal_insult;Remove Kebab insult;;;;;;;;;;;x
```

Comment:
lel, ok :D I forgot that Swedes are politically correct. It seems that this modifier isn't in use, so I removed it completely.